src/util.js

```diff
@@ -0,0 +1,10 @@
+export function dequote (input) {
+  input = input.trim()
+
+  const singleQuote = input.startsWith('\'') && input.endsWith('\'')
```

Comment:
Immutable is preferable...
```js
function dequote(input) {
  const output = input.trim();

  // Check for 3 valid quote styles at beginning and end of string.
  if (/(?:^'.*'$)|(?:^".*"$)|(?:^`.*`$)/.test(output) {
    return output.slice(1, -2);
  }
  return output;
}

export { dequote };
```

Now, that regex is an abomination, but it checks for all 3 types of quotes, pinned to start and end, and is very compact.